docs/registries.md

```diff
@@ -16,16 +16,22 @@ To support private GitHub, simply authenticate with your private GitHub account:
 ```
 
 ```
-Would you like to set up your GitHub credentials? [yes]: 
+Would you like to set up your GitHub credentials? [yes]:
      If using two-factor authentication or to avoid using your password you can generate an access token at https://github.com/settings/applications.
 
 Enter your GitHub username: username
-Enter your GitHub password or access token: 
-Would you like to test these credentials? [yes]: 
+Enter your GitHub password or access token:
+Would you like to test these credentials? [yes]:
 ```
 
 This will enable private repo installs.
 
+#### Github Rate Limiting
+
+Github rate limits by IP unless an auth token is provided. On hosted services such as Circle, Travis or Heroku where you share IPs with other users also using Github you will likely hit these limits. Setting a `JSPM_GITHUB_AUTH_TOKEN` variable in your environment is enough to tell jspm to use it. Most services have the ability to do this from their web panels.
```

Comment:
Yeah, I think it makes sense to say what happens internally when the env variable is encountered.